request_logger.py

```
@@ -0,0 +1,105 @@
+logging.basicConfig(
+    filename=log_file,
+    level=logging.DEBUG,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger('http_logger')
+
+def monkey_patch_requests():
+    """
+    Patch the requests library to log all requests and responses
+    """
+    import requests
```

Ruff (0.8.2)

`requests` imported but unused

Remove unused import: `requests`

(F401)